SimPEG/Optimization.py

```diff
@@ -892,7 +892,7 @@ def __init__(self, **kwargs):
     tolCG = 1e-1
 
     stepOffBoundsFact = 0.1 # perturbation of the inactive set off the bounds
-
+    stepActiveset = True
```

Comment:
interesting, you would set this to false if you don't want to change the active / inactive sets? Do you have an example that uses this?